src/main/java/de/rabea/HttpVerb.java

```diff
@@ -8,26 +8,12 @@
     HEAD;
 
     public static HttpVerb convert(String word) {
-        if (word.equals("GET")) {
-            return GET;
-        } else if (word.equals("POST")) {
-            return POST;
-        } else if (word.equals("PUT")) {
-            return PUT;
-        } else if (word.equals("OPTIONS")) {
-            return OPTIONS;
-        } else if (word.equals("HEAD")) {
-            return HEAD;
+        HttpVerb[] verbs = HttpVerb.class.getEnumConstants();
+        for (HttpVerb verb : verbs) {
```

Comment:
ok, thanks, I'll use that instead! I've changed it on my current branch, so won't update this PR.